django_netjsonconfig/migrations/0039_vpn_format_dh.py

```diff
@@ -0,0 +1,23 @@
+from django.db import migrations
+
+
+def format_dh(apps, schema_editor):
+    Vpn = apps.get_model('django_netjsonconfig', 'Vpn')
+    byte_dh_beginning = "-----BEGIN DH PARAMETERS-----\nb'"
+    byte_dh_end = "'\n-----END DH PARAMETERS-----\n"
+
+    for vpn in Vpn.objects.all():
+        if vpn.dh.startswith(byte_dh_beginning) and vpn.dh.endswith(byte_dh_end):
```

Comment:
this is the DH value I noticed in existing instances:
```
"b'-----BEGIN DH PARAMETERS-----\\nMIGHAoGBAKQku25Xa+mkCz264wbefgyFIt3ir1vR1giDcx9LvK4ml7mNywD01HaK\\nL6X8aYY4hmZN4B0S
j/quTNKx0BaFLK6nhCH2LIZl2FaR9KV1i7caXM4mHNn6gCID\\nWtuSpjETCa7TRHF+AYOeOAc00vg3+1dAxKNYx4M8IwqNKxCES5prAgEC\\n-----END
 DH PARAMETERS-----\\n'"
```

I think the current patch doesn't fix it. I suggest to test with that value.